test/ErrorMessage.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import ErrorMessage from "../src/components/ErrorMessage";
+
+describe('should render ErrorMessage component', () => {
+  const element = mount(<ErrorMessage message="Test message" />);
```

Comment:
Try using `shallow` whenever possible - it will help you testing on unit level and is a bit faster.

Reply:
Changed.